### PR TITLE
NOTIF-475 UX improvements on user preferences

### PR DESCRIPTION
**Pull request by aferd**

### CODE REVIEW

**Comment by aferd** [comment]
Screen Shot 2022-02-22 at 2 05 53 PM

File: src/PresentationalComponents/Notification/Notification.js [hunk 1]
```diff
@@ -109,20 +113,25 @@ const Notification = () => {
   return (
     <React.Fragment>
       <PageHeader>
-        <PageHeaderTitle title="Notification Insights" />
+        <Split>
+          <SplitItem isFilled>
+            <PageHeaderTitle
+              title={`My Notifications | ${bundleDisplayTitle.title}`}
```
Comment (josejulio) [inline src/PresentationalComponents/Notification/Notification.js:119]: don't forget to update this one too
```suggestion
              title={`My Notifications | ${bundleDisplayTitle}`}
```

File: src/PresentationalComponents/Notification/Notification.js [hunk 2]
```diff
@@ -109,20 +113,25 @@ const Notification = () => {
...
+            />
+            <StackItem>
+              This service allows you to opt-in and out of recieving
+              notifications. Your Organization Administrator has configured
+              which notifications you can or can not recieve in their
+              <a href={`/settings/notifications/${bundleName}`}> Settings</a>.
```
Comment (josejulio) [inline src/PresentationalComponents/Notification/Notification.js:125]: We need to take into account the beta/stable status here. Not sure how it's done here, but you can search for `beta` and see how is handled.

File: src/actions.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { getApplicationSchema, saveValues as save } from './api';
 import { ACTION_TYPES } from './constants';
 import config from './config/config.json';
 
-const notificationConfigForBundle = (bundleName) =>
+export const notificationConfigForBundle = (bundleName) =>
```
Comment (josejulio) [inline src/actions.js:5]: So, for the sake of having code where it belongs, I think we should move out this function to a separate place. 

This file have the redux actions, maybe put it on utilities: https://github.com/RedHatInsights/user-preferences-frontend/blob/master/src/Utilities/functions.js

**Review by josejulio — CHANGES_REQUESTED** [review]

**Comment by josejulio** [comment]
@aferd there is one failing test - looks like one import could be wrong?

**Comment by josejulio** [comment]
@karelhala could you take a look? thanks!

File: src/PresentationalComponents/Notification/Notification.js [hunk 4]
```diff
@@ -109,20 +111,29 @@ const Notification = () => {
   return (
     <React.Fragment>
       <PageHeader>
-        <PageHeaderTitle title="Notification Insights" />
+        <Split>
+          <SplitItem isFilled>
+            <PageHeaderTitle
+              title={`My Notifications | ${bundleDisplayTitle}`}
+              style={{ paddingBottom: '8px' }}
```
Comment (karelhala) [inline src/PresentationalComponents/Notification/Notification.js:118]: Can we move this to `scss` file with class?

File: src/PresentationalComponents/Notification/Notification.js [hunk 5]
```diff
@@ -109,20 +111,29 @@ const Notification = () => {
...
+            />
+            <StackItem>
+              This service allows you to opt-in and out of receiving
+              notifications. Your Organization Administrator has configured
+              which notifications you can or can not receive in their
+              <a href={getBaseName(`/settings/notifications/${bundleName}`)}>
```
Comment (karelhala) [inline src/PresentationalComponents/Notification/Notification.js:124]: We should use `chrome`'s api similiar to how dashboard is doing it https://github.com/RedHatInsights/insights-dashboard/blob/master/src/SmartComponents/Drift/DriftCard.js#L43 this way we won't force page refresh, but rather make it SPA like.

File: src/PresentationalComponents/Notification/Notification.js [hunk 6]
```diff
@@ -32,16 +31,19 @@ import FormButtons from '../shared/FormButtons';
...
 import { addNotification } from '@redhat-cloud-services/frontend-components-notifications/redux';
+import getBaseName from '../../Utilities/getBaseName';
 
 const Notification = () => {
   const { bundleName } = useParams();
   const dispatch = useDispatch();
   const store = useSelector(
     ({ notificationPreferences }) => notificationPreferences
   );
+  const bundleDisplayTitle = notificationConfigForBundle(bundleName).title;
```
Comment (karelhala) [inline src/PresentationalComponents/Notification/Notification.js:46]: You should use safe accessor here
```suggestion
  const bundleDisplayTitle = notificationConfigForBundle(bundleName)?.title;
```

**Review by karelhala — COMMENTED** [review]
Looking good, just a few minor changes and we are golden.

**Pull request merged**